### ISSUE TITLE
Question about the loss

### ISSUE THREAD

**Issue by DavisonHu** [Amir-Arsalan/Synthesize3DviaDepthOrSil#6]
Hello,I read the paper "Synthesizing 3D Shapes via Modeling Multi-view Depth Maps and Silhouettes with Deep Generative Networks",the loss function which mention inside is to minimize :                                     loss = -DKL(q(Z|X)||p(Z)) + E[logp(x|Z)] 
but the loss function in VAE is :
loss =   DKL(q(Z|X)||p(Z)) - E[logp(x|Z)] 
I am confused about it, I hope you can give me some advice,thank you

**Comment by Amir-Arsalan** [comment]
They are essentially the same. You always optimize the parameters by going in the negative direction of the gradients, hence we put the negative sign there.

**Comment by Amir-Arsalan** [comment]
@DaviosnHu [This](https://stackoverflow.com/questions/45564706/why-the-loss-of-variational-autoencoder-in-many-implementations-have-opposite-si) will clarify things for you.
